build.xml

```diff
@@ -179,7 +179,7 @@
     <copy todir="war/WEB-INF/lib" file="${gwt.sdk}/gwt-servlet.jar" />
     <copy todir="war/WEB-INF/lib" file="${gwt.sdk}/gwt-servlet-deps.jar" />
     <!-- Add any additional server libs that need to be copied -->
-    <copy todir="war/WEB-INF/lib" file="lib/jnr-fuse-0.5.3-all.jar" />
+    <copy todir="war/WEB-INF/lib" file="lib/jnr-fuse-0.3-all.jar" />
```

Comment:
A new version was released on 1 April. 0.5.4 https://github.com/SerCeMan/jnr-fuse/releases

Reply:
We had a new version checked in as well as 0.3, but it doesn't work. So until someone has time to fix it I've removed it.